libraries/app/database_api.cpp

```diff
@@ -1840,9 +1840,6 @@ set<public_key_type> database_api_impl::get_potential_signatures( const signed_t
          const auto& auth = id(_db).active;
          for( const auto& k : auth.get_keys() )
             result.insert(k);
-         // Also insert owner keys since owner can authorize a trx that requires active only
-         for( const auto& k : id(_db).owner.get_keys() )
-            result.insert(k);
```

Comment:
Why is this removed? According to the documentation,
> This method will return the set of all public keys that could possibly sign for a given transaction.

That includes the owner keys of "active" authorities, doesn't it?

Reply:
Owner keys **of immediately required accounts** are implicitly included due to another change in same commit:
```
-      s.check_authority( active  );
+      s.check_authority( active ) || s.check_authority( get_owner( active ) );
```

Owner keys **of non-immediately required accounts** are unable to authorize the transaction due to [the behavior you mentioned below](https://github.com/bitshares/bitshares-core/pull/570#issuecomment-357867908).

Reply:
You're right, thanks. I misunderstood the purpose of the API call.